lib/utils/format.js

```diff
@@ -419,6 +419,10 @@
         var className = /*'unipika-' + */node.$type.replace('.', '_');
         var title = '';
 
+        if (node.$category) {
+            className += WHITESPACE + 'pg_category_' + /*'unipika-' + */node.$category.toLowerCase()
```

Comment:
It brings XSS vulnerability to the code, the following value of category is dangerous:
```ts
{
   $value: '...',
   $type: '...',
   $category: `"/><script>alert("hello XSS")</script><span className="`
}
```
Do we know all possible categories? If so we can register separate plugin for each category and avoid it.

Reply:
We know valid categories, but I'm hot sure we'll use them further to format values. May be we'll use types. So it seems that adding a plugin for every category is redundant for now. I'd better add validation to avoid unwanted values.